package.json

```
@@ -32,9 +32,10 @@
   "homepage": "https://github.com/UKHomeOffice/asl-internal-ui#readme",
   "dependencies": {
     "@asl/pages": "^31.8.1",
```

```suggestion
    "@asl/pages": "^31.8.2",
```
+ re-run npm install and push new lock file as well.